modules/sdk/src/main/scala/org/tessellation/sdk/package.scala

```diff
@@ -47,4 +56,24 @@ package object sdk {
       EventTrigger.getClass -> 530,
       TimeTrigger.getClass -> 531
     ).union(sharedKryoRegistrar)
+
+  object PriorityPeerIds {
```

Comment:
What about testnet? Also, could you make a ticket with high priority for moving it to env or build configuration (as you want to move it out from the code in different PR)?